src/worldfactory.cpp

```diff
@@ -767,12 +772,14 @@ void worldfactory::show_active_world_mods( const std::vector<std::string> &world
 
         if( action == "UP" ) {
             cursor--;
-            if( cursor < 0 ) {
+            // If it went under 0, loop back to the end of the list.
+            if( cursor > num_mods -1 ) {
```

Comment:
Again, triggering unsigned underflow is not a valid approach, if a variable is going to be subtracted below zero, it should be signed.